libs/node-custom/src/mutex.ts

```diff
@@ -0,0 +1,30 @@
+import { Optional } from '@votingworks/types';
+
+/**
+ * A simple mutex implementation.
+ */
+export class Mutex {
+  private locked = false;
```

Comment:
Is there a distinction between `Mutex` and `Lock`?

Reply:
This is sort of what I meant in the description how the locking is messed up. It's kind of in transition right now. `Lock` waits when `acquire` is called, `Mutex` just fails if it's busy. The latter emulates how the official Custom driver works. I haven't decided which approach to take.